src/clients/music/mpris.rs

```diff
@@ -35,34 +39,36 @@ impl Client {
                 // D-Bus gives no event for new players,
                 // so we have to keep polling the player list
                 loop {
-                    let players = player_finder
-                        .find_all()
-                        .expect("Failed to connect to D-Bus");
+                    // mpris-rs does not filter NoActivePlayer errors, so we have to do it ourselves
+                    let players = player_finder.find_all().unwrap_or_else(|e| match e {
+                        mpris::FindingError::DBusError(DBusError::TransportError(
+                            transport_error,
+                        )) if transport_error.name() == Some(NO_ACTIVE_PLAYER)
+                            || transport_error.name() == Some(NO_REPLY) =>
+                        {
+                            Vec::new()
+                        }
```

Comment:
This can be squashed down a bit:

```suggestion
                        )) if transport_error.name() == Some(NO_ACTIVE_PLAYER)
                            || transport_error.name() == Some(NO_REPLY) => vec![]
```

Reply:
This was a bit finicky. Tried you suggestion and other changes for a bit but cargo fmt keeps adding the extra lines with {}. Tried shortening the previous line using:
```
use mpris::EventError::DBusError as EventDBusError;
use mpris::FindingError::DBusError as FindingDBusError;
```
but wont pass cargo fmt unless I make it something like EDE and FDE for the above errors which sacrifices clarity for brevity.
Please lmk if you have something that passes cargo fmt. Thank you!

Reply:
Ah fair enough, I wonder if it's just because of the pattern complexity it demands a block. I'm not particularly bothered by that either way so happy to mark resolved.